lib/hex_web/util.ex

```diff
@@ -20,6 +20,12 @@ defmodule HexWeb.Util do
     :calendar.gregorian_days_to_date(today_days - 1)
   end
 
+  def last_year do
+    {today, _time} = :calendar.universal_time()
+    today_days = :calendar.date_to_gregorian_days(today)
+    :calendar.gregorian_days_to_date(today_days - 35)
+  end
```

Comment:
This is only 35 days back, so it shouldn't be called last_year. It's only used in the sample_data script so move it there.